packages/flex-plugin-e2e-tests/src/utils/console-api.ts

```diff
@@ -0,0 +1,144 @@
+import { stringify } from 'querystring';
+
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { logger } from 'flex-plugins-utils-logger';
+
+const csrfTokenRegex = new RegExp(/name="CSRF"\svalue="(.*?)"/m);
+const twVisitorCookieRegex = new RegExp(/tw-visitor=(.*?);/m);
+const identityCookieRegex = new RegExp(/identity=(.*?);/m);
+const serverIdentityCookieRegex = new RegExp(/server-identity=(.*?);/m);
+
+interface ConsoleAuthOptions {
+  email: string | undefined;
+  password: string | undefined;
+}
+
+const SET_COOKIE = 'set-cookie';
+
+export enum Cookie {
+  visitor = 'tw-visitor',
+  sIdentity = 'server-identity',
+  identity = 'identity',
+}
+
+export interface Cookies {
+  [Cookie.identity]: string | undefined;
+  [Cookie.sIdentity]: string;
+  [Cookie.visitor]: string;
+}
+
+/**
+ * Defines interactions with Console API
+ */
+export class ConsoleAPI {
+  private baseUrl: string;
+
+  private consoleAuthOptions: ConsoleAuthOptions;
+
+  /**
+   * Initialises console base url and verifies console credentials
+   * @param consoleBaseUrl base url of Twilio console
+   */
+  constructor(consoleBaseUrl: string) {
+    this.baseUrl = consoleBaseUrl;
+
+    if (!process.env.CONSOLE_EMAIL || !process.env.CONSOLE_PASSWORD) {
```

Comment:
Can you process the env and add it to the testParams https://github.com/twilio/flex-plugin-builder/blob/main/packages/flex-plugin-e2e-tests/src/index.ts#L41? Let's not use `process.env*` at all anywhere else